templates/cli/appwrite.json.twig

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://raw.githubusercontent.com/ScoopInstaller/Scoop/master/schema.json",
+    "version": "{{ sdk.version }}",
+    "description": "Appwrite cli project manager",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/{{ sdk.gitUserName }}/{{ sdk.gitRepoName | caseDash }}/releases/download/{{ sdk.version }}/{{ language.params.executableName }}-cli-win-x64.exe",

```

Comment:
Scoop let's you specify architecture as well. It would probably be good to differentiate between arm and x86.

Reply:
Nope. The architecture difference is x32 vs x64. This is addressed in issue ScoopInstaller/Scoop#3146, however, that is not fixed yet. I am subscribed to it though so as soon as they add support for ARM I will update the manifest to reflect those changes.